src/components/HURUmap/PanelButtonGroup/index.js

```diff
@@ -0,0 +1,65 @@
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+function PanelButtonGroup({
+  items,
+  value,
+  handleChange: handleChangeProps,
+  ...props
+}) {
+  const classes = useStyles(props);
+
+  const handleChange = (_, selected) => {
+    handleChangeProps(selected);
+  };
+
+  if (!items?.length) {
+    return null;
+  }
+
+  return (
+    <div className={classes.root}>
+      <ToggleButtonGroup
+        orientation="vertical"
+        value={value}
+        exclusive
+        onChange={handleChange}
+      >
+        {items.map((buttonProps) => (
```

Comment:
```suggestion
        {items.map(({icon, ...buttonProps}) => (
```
Or why are we passing `icon` to `ToggleButton`?